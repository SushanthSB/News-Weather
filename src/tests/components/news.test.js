import React from 'react';
import News from '../../components/news'
import renderer from 'react-test-renderer'

describe('News()',() => {
	let props = {
		fetch:{"data":[{"title":"News"},{"description":"Dispay news"},{"url":"https://newsapi.or"}]}
	}
	it('should render without fail',() => {
		const tree = renderer.create(<News {...props} />).toJSON
		expect(tree).toMatchSnapshot();
	})
})