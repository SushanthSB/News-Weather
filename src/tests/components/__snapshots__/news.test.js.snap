// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News() should render without fail 1`] = `[Function]`;
